src/lib.rs

```diff
@@ -10,13 +10,17 @@ mod tests;
 mod types;
 mod formatter;
 mod fmtstr;
-mod fmtf64;
-mod fmti64;
+
+#[macro_use]
+mod fmtnum;
 
 pub use types::{Result, FmtError, Alignment, Sign};
 pub use fmtstr::strfmt_map;
 pub use formatter::Formatter;
 
+// u128 & i128 unstable (see https://github.com/rust-lang/rust/issues/35118)
+fmtint!(u8 i8 u16 i16 u32 i32 u64 i64 usize isize);
```

Comment:
whoa! I didn't know this was possible, I thought they would have to each be their own line. Good work!